### PR TITLE
TEST: Run provider tests on pre-merge

### PR DESCRIPTION
**Pull request by jfharden**
## WHAT YOU DID
_A brief description of the pull request:_

## How to test

- How should it be reviewed? 
- What feedback do you need? 
- If manual testing is needed, give suggested testing steps.

### Logging

- [ ] only emit log lines at ERROR level which require immediate attention from a support engineer. These will trigger a zendesk alert.

### Documentation

- [ ] Updated README.md for any of the following ?

* Introduced any new environment variables / removed existing environment variable
* Added new API / updated existing API definition

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -35,3 +35,9 @@ jobs:
           docker pull govukpay/postgres:11.1
       - name: Run unit and integration tests
         run: mvn clean verify
+
+  provider-contract-tests:
+    uses: alphagov/pay-ci/.github/workflows/_run-provider-contract-tests.yml@TEST-pact-tests-pre-merge
```
Comment (jfharden) [inline .github/workflows/run-tests.yml:40]: This doesn't need the @ branch anymore, it works on the master version

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -35,3 +35,9 @@ jobs:
           docker pull govukpay/postgres:11.1
       - name: Run unit and integration tests
         run: mvn clean verify
+
+  provider-contract-tests:
```
Comment (jfharden) [inline .github/workflows/run-tests.yml:39]: I think this might need to come after unit and integration tests since that verifies the pacts

**Comment by katstevens** [comment]
@jfharden I think we can close this PR for now?

**Comment by katstevens** [comment]
@jfharden going to close this PR - we can always reopen/rebase if needed

**Pull request closed without merging**